src/block/posts/index.php

```diff
@@ -115,7 +115,8 @@ function generate_render_item_from_stackable_posts_block( $post, $attributes, $t
 			} else {
 				$excerpt = implode( ' ', $excerpt );
 			}
-
+			// Trim the excerpt differently if there are chinese characters.
+			$excerpt = filter_cjk_characters( $excerpt, $trim_to_length );
```

Comment:
While this would technically work, the logic is off. The CJK trimming shouldn't be called after the if/else statement, because this would mean that the strings will always be trimmed and will be trimmed twice. 

Our logic above is as follows (in the `if` statement):
1. If the excerpt is too long, trim it
2. If not, just use as-is

The logic in the PR is:
1. If the excerpt is too long, trim it
2. If not, just use as-is
3. After 1 & 2, trim CJK characters

The trimming of CJK should make the logic be:
1. If the excerpt has CJK characters, trim it if it's too long,
2. If the excerpt doesn't have CJK characters but it's too long, trim it
4. If not, just use as-is

Reply:
Got it.